pubspec.yaml

```diff
@@ -2,6 +2,8 @@ name: instabug_flutter_modular
 description: An add-on for the Instabug Flutter SDK that provides screen loading support for Flutter Modular v5.
 version: 0.0.1
 homepage: https://www.instabug.com
+# TODO: remove publish_to: after changing instabug_flutter dependency  to the production version
+publish_to: none
```

Comment:
i think we need to change it now

Reply:
Removing it now causes the Flutter analyzer job to fail due to the git dependency. Therefore, as the todo comment suggests, I recommend leaving it as is until we update the dependencies with the production version.